torch/testing/_internal/opinfo/definitions/linalg.py

```diff
@@ -1703,6 +1703,15 @@ def make_input():
                 device_type="cpu",
                 dtypes=(torch.long,),
             ),
+            # RuntimeError: The tensor has a non-zero number of elements, but
+            # its data is not allocated yet. Caffe2 uses a lazy allocation, so
+            # you will need to call mutable_data() or raw_mutable_data() to
+            # actually allocate memory.
+            DecorateInfo(
+                unittest.skip("Skipped!"),
+                "TestCommon",
+                "test_out",
+            ),
```

Comment:
Is this okay? Had to add a skip after adding `SymInt` support due to the error above. Unfortunately, it's in the `out` test.